Add decode to build_paper output to make it format better

ksunden opened the pull request:
Was printing Byte-string, which was horrible to read

ksunden committed:
Add decode to build_paper output to make it format better (was byte string)

mpacer commented on publisher/build_paper.py:
```diff
@@ -149,7 +149,7 @@ def tex2pdf_singlepass(out_path):
         print("PDFLaTeX error output:")
         print("=" * 80)
-        print(out)
+        print(out.decode())
```
My intuition is that we should decode it to `'utf-8'` for consistency with Python3 (which is currently planned to be the only supported python as of next year). Is there a reason not to do that?

mpacer commented:
🎉 For making this PR against `dev`. You rock!

deniederhut commented:
I would like to second to motion for you rocking.

ksunden committed:
Add 'utf-8'

Also noted that `err` is also printed, added decode to maintain consistency there

deniederhut commented:
![1pfr](https://user-images.githubusercontent.com/6868396/46472648-7917ce00-c7a3-11e8-9f77-742f9b95bf9d.gif)
